.github/workflows/prune-images.yml

```diff
@@ -0,0 +1,63 @@
+name: Prune Images
+run-name: Prune old images from OpenShift
```

Comment:
It would be recommended add some inputs to the `run-name` to differentiate them easily.
For instance:
```
Release - Deploy all from ${{ inputs.gitRef }} to ${{ inputs.environment }}
```